cluster-provision/README.md

```diff
@@ -16,16 +16,16 @@
 
 * `kubevirtci/cli`: `sha256:1dd015dea4f12e6dcb8e31be3eeb677fed96f290ef4a4892a33c43d666053536`
 * `kubevirtci/gocli`: `sha256:1e21a7b0fc959ae2a47d1d6462ceb74f9e9181d52f9bc618e61bcab80bedaa9e`
-* `kubevirtci/base`: `sha256:034de1a154409d87498050ccc281d398ce1a0fed32efdbd66d2041a99a46b322`
-* `kubevirtci/centos:1905_01`: `sha256:f2b204a3fabf71494c3cf7a145dae88e63f068a84ca10149b0aec523045dc2c1`
-* `kubevirtci/os-3.11.0-multus`: `sha256:f8de6bda57209553e1a9e428dca1432270b7db0e5206376898b93aea64afd053`
+* `kubevirtci/base`: `sha256:59f2a244d7f2b966f9f1b8a9b4d9286b779027f774201514c4f27afd0e7bc585`
+* `kubevirtci/centos:1905_01`: `sha256:a02a1253a4bb734ee7fb244b42afcc7fdc66f5e26f440eb718876f1913e1717a`
+* `kubevirtci/os-3.11.0-multus`: `sha256:9b6f0f8535a82515960cc48bc68a49857206dfe57426d3ad383411d7752b1c8c`
 * `kubevirtci/os-3.11.0:`: `sha256:527daa78a9e460053925a862324f67390d45ffacaf26fe3fd5a6dc20a7892534`
```

Comment:
seems you forgot os-3.11.0, should be sha256:fb5d586c62887f2f5e1ddeeae05da94c28baf4bcc27201e543afee3984b430b0